### PR TITLE
Mention new supported frameworks

### PR DESCRIPTION
**Pull request by Hypnosphi**

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ That way you can write your stories how they are best, and preview them how you
 
 We believe in the power of react, and think it's the right choice for a lot of projects.
 But it's up to you and your team to decide your stack.
-Unfortunately if you choose anything other then React or React-Native you can not use storybook.
+Unfortunately if you choose anything other then React or React-Native or Vue or Angular you can not use storybook.
```
Comment (igor-dv) [inline ROADMAP.md]: I think somewhere should be a list of supported frameworks, and we will be able to refer to it.

Comment (Hypnosphi) [reply ROADMAP.md]: What would be a good place for that?

Comment (igor-dv) [reply ROADMAP.md]: Maybe the main README? Or a separate `FRAMEWORKS.md` ? I also wanted to push this 
https://gist.github.com/igor-dv/d621819b0d772aea0df88608f74e4da3
somewhere.., So maybe to the same place =)

**Review by igor-dv — APPROVED** [review]
👍

**Pull request merged**